cylc/flow/prerequisite.py

```diff
@@ -104,12 +103,6 @@ class Prerequisite:
     SATISFIED_TEMPLATE = 'bool(self._satisfied[("%s", "%s", "%s")])'
     MESSAGE_TEMPLATE = r'%s/%s %s'
 
-    DEP_STATE_SATISFIED: SatisfiedState = 'satisfied naturally'
-    DEP_STATE_SATISFIED_BY = 'satisfied by {} mode'
-    DEP_STATE_OVERRIDDEN = 'force satisfied'
-    DEP_STATE_UNSATISFIED = False
-    SATISFIED_MODE_RE = re.compile(r'satisfied by .* mode')
```

Comment:
Most of these unused, some re-added after being removed in https://github.com/cylc/cylc-flow/pull/6293